manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: "EQAditu.AdvancedCombatTracker"
+PackageVersion: "3.5.0.273"

```

Comment:
Version is blank in ARP (Programs and Features) could you contact the dev about this?

Reply:
Tracking in EQAditu/AdvancedCombatTracker#33